src/django_components/util/tag_parser.py

```diff
@@ -625,10 +757,15 @@ def extract_spread_token(curr_struct: TagValueStruct, filter_token: Optional[str
                 # Get past the filter tokens like `|` or `:`, until the next value part.
                 # E.g. imagine:    `height="20" | yesno : "1,2,3" | lower`
                 # and we're here:               ^
+                # (or here)                             ^
+                # (or here)                                       ^
                 # and we want to parse `yesno` next
                 if not is_first_part:
                     filter_token = taken_n(1)  # | or :
                     take_while(TAG_WHITESPACE)  # Allow whitespace after filter
+
+                    if filter_token == ":" and values_parts[-1].filter != "|":
```

Comment:
Django's filters have structure `var|filter:arg`, where the filters can be chained. This addition catches the case when the expression uses `:` without previously using `|`.

So e.g. `var|filter:arg` is fine, while `var:arg` will raise an error.